### PR TITLE
Update definition of menu_bar

### PR DESCRIPTION
**Pull request by Bachibouzouk**
- fix typo in attribute name
- jkdasd
- sadhask

Fix #Issue

**Changes proposed in this pull request**:
- Your_changes

The following steps were realized, as well (if applies):
- [ ] Apply black (`black . --exclude docs/`)
- [ ] Update the CHANGELOG.md

The following optional steps were realized:
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: explain in readthedocs
- [ ] Write test(s) for your new patch of code

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by FredericHirschmueller** [comment]
Hello

File: docs/view_components/menu_bar.rst [hunk 1]
```diff
@@ -38,15 +38,15 @@ Attributes
 Actions
 ^^^^^^^
 
-* Clicking on any of the menus' option results in a drop-down list of sub-menu's options
+* ysdkljasdkasldjasldkad alkd
```
Comment (FHell) [inline docs/view_components/menu_bar.rst]: Should be ;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv

Comment (c-moeller) [inline docs/view_components/menu_bar.rst]:
```suggestion
* sdfsdfsdfsdfsdfsdfsdf
```

File: docs/view_components/menu_bar.rst [hunk 2]
```diff
@@ -38,15 +38,15 @@ Attributes
 Actions
 ^^^^^^^
 
-* Clicking on any of the menus' option results in a drop-down list of sub-menu's options
```
Comment (FredericHirschmueller) [inline docs/view_components/menu_bar.rst:41]: I like these changes

File: docs/view_components/menu_bar.rst (hunk 1 above)
Comment (Bachibouzouk) [reply docs/view_components/menu_bar.rst]:
```suggestion
;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv
```

File: docs/view_components/menu_bar.rst [hunk 3]
```diff
@@ -67,6 +67,6 @@ Link with other view-components
 Rendering of the view-component
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* Rectangular drop-down menu buttons present on a horizontal bar above the flow chart ribbon
+* Rectanguladasdasdadadadada above the flow chart ribbon
```
Comment (c-moeller) [inline docs/view_components/menu_bar.rst]:
```suggestion
* above the flow chart ribbon
```

**Review by c-moeller — CHANGES_REQUESTED** [review]
Great! But please consider some changes.

File: docs/view_components/menu_bar.rst [hunk 4]
```diff
@@ -38,15 +38,15 @@ Attributes
...
+* ysdkljasdkasldjasldkad alkd
 * Clicking on *File* would show a drop-down list with the following submenus or options:
     I. Load project
     II. New project
     III. Save project
     IV. Save project as
     V. Export project
     VI. Exit open_plan
-* clicking on *Preferences* would show a drop-down list with the following submenus or options:
+* clicking on *Preferences* would show aadasdsaddasdd ist with the following submenus or options:
```
Comment (srhbrnds) [inline docs/view_components/menu_bar.rst]: Dropdown lists are not handy 
```suggestion
* clicking on *Preferences* would show pop-up window with the following submenus or options:
```

File: docs/view_components/menu_bar.rst (hunk 2 above)
Comment (FredericHirschmueller) [inline docs/view_components/menu_bar.rst:41]: Hello

File: docs/view_components/menu_bar.rst [hunk 5]
```diff
@@ -38,15 +38,15 @@ Attributes
 Actions
 ^^^^^^^
 
-* Clicking on any of the menus' option results in a drop-down list of sub-menu's options
+;askdjf;lkadhf;lkadjhfl;ahjdgo;iahjwoivrnavj;cjvk;ajsdf;ojv;oiwjdv
```
Comment (Bachibouzouk) [inline docs/view_components/menu_bar.rst]: wssdfsf

File: docs/view_components/menu_bar.rst [hunk 6]
```diff
@@ -38,15 +38,15 @@ Attributes
...
 * Clicking on *File* would show a drop-down list with the following submenus or options:
     I. Load project
     II. New project
```
Comment (PoetzschJohannes) [inline docs/view_components/menu_bar.rst:44]: Hallo Test

**Review by Bachibouzouk — COMMENTED** [review]
I think this is overall great but you should take the stakeholderrs into account more in the future

File: docs/view_components/menu_bar.rst (hunk 5 above)
Comment (Bachibouzouk) [reply docs/view_components/menu_bar.rst]: askjdsadkah

Comment (FredericHirschmueller) [reply docs/view_components/menu_bar.rst]: hello

Comment (Bachibouzouk) [reply docs/view_components/menu_bar.rst]:
```suggestion
;askdjf;lkadhf;lkadjhfl;ahjdgo;
```

**Pull request closed without merging**